README.md

```diff
@@ -27,6 +27,14 @@ The `ui-eholdings` module adds the ability to manage electronic holdings in FOLI
 * `yarn start`
 * Visit your app at [http://localhost:8080](http://localhost:8080).
 
+On Windows: 
+If you encounter problems trying to run your app on Windows, specifically, with NODE_ENV:
+
+* Set environment variable NODE_ENV=development
+  `SET NODE_ENV=development` if running from Windows command prompt
+  `export NODE_ENV=development` if running from Cygwin on Windows
+* Edit ui-eholdings/package.json to remove `NODE_ENV=development` from scripts object
```

Comment:
If it's causing trouble on Windows, should we remove `NODE_ENV=development` from the `start` script completely? My copy on macOS seems to work fine without it.

Reply:
The CLI defaults NODE_ENV to "development" for the `serve` command.
https://github.com/folio-org/stripes-cli/blob/0b939ac8951a3146b940b05b7d591edc39eefc30/lib/commands/serve.js#L11-L15

Reply:
Do you have the environment variable of NODE_ENV set and just got rid of it from package.json?
I see that NODE_ENV is being used in .stripesclirc.js

Reply:
I don't seem to be setting `NODE_ENV` anywhere. Maybe it's "development" by default? I opened a PR here for us to test it out: https://github.com/folio-org/ui-eholdings/pull/249

Reply:
tested your PR on both Windows and Linux and it works just fine. However, point to note, is that after we get that change, folks have to force install their dependencies again, either yarn or yarn install. Else its giving problems finding stripes in Windows. 
Another point is would we have any reason to set NODE_ENV to something other than "development"? If that is the case, should we mention in README that folks can set that env variable?